packages/appstore/src/components/appstore-banner/svg-migration-banner.tsx

```diff
@@ -2,17 +2,24 @@ import React from 'react';
 import { Button, Text } from '@deriv/components';
 import { Localize } from '@deriv/translations';
 import { observer, useStore } from '@deriv/stores';
+import { useMT5SVGEligibleToMigrate } from '@deriv/hooks';
 import AppstoreBannerImage from './appstore-banner-image';
 
 const SVGMigrationBanner = observer(() => {
     const {
         ui,
         modules: { cfd },
     } = useStore();
+    const {
+        eligible_svg_to_bvi_derived_accounts,
+        eligible_svg_to_bvi_financial_accounts,
+        eligible_svg_to_vanuatu_derived_accounts,
+        eligible_svg_to_vanuatu_financial_accounts,
+    } = useMT5SVGEligibleToMigrate();
     const { is_mobile, toggleMT5MigrationModal } = ui;
     const { setMT5MigrationError } = cfd;
-    const is_deriv_mt5 = true;
-    const is_financial_mt5 = true;
+    const is_deriv_mt5 = eligible_svg_to_bvi_derived_accounts || eligible_svg_to_vanuatu_derived_accounts;
+    const is_financial_mt5 = eligible_svg_to_bvi_financial_accounts || eligible_svg_to_vanuatu_financial_accounts;
```

Comment:
@suisin we need to show the content in banner based on the accounts to migrate.( line 42). can you check this?
![Screenshot 2023-09-21 at 12 49 25 PM](https://github.com/amina-deriv/deriv-app/assets/84661147/6f3d5d0f-6291-44d9-bcf9-2beca45eb2e1)

Reply:
Yeapp the current condition will change the content of the banner based on the responded value. If both of the value for `is_deriv_mt5` and `is_financial_mt5` are true, then it will display the longer text for the banner, else it will only display either one of the `MT5 Derived SVG` or `MT5 Financial SVG` message.

Reply:
oh i got confused with variable naming.
can we rename  `is_deriv_mt5` to has_derived_mt5_to_migrate  and is_deriv_financial_mt5 to has_financial_mt5_to_migrate